README.md

```diff
@@ -201,6 +223,128 @@ Sample request after deploying.
 curl https://xxxxxx.execute-api.us-east-1.amazonaws.com/dev/sns -d '{"message": "testtest"}' -H 'Content-Type:application/json'
 ```
 
+### DynamoDB
+
+The (DynamoDB Operation)[https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_Operations.html] is automatically mapped by the `method` with following HTTP specification if you omit the `action`.
+
+| Method | Mapped action | Note |
+----|----|----
+| post | PutItem | hashkey is set apigateway requestid automatically by default |
+| put | PutItem | hashkey is set `pathParam` or `queryStringParam` |
+| patch | UpdateItem ||
+| get | GetItem ||
+| delete | DeleteItem ||
+
+Sample syntax for DynamoDB proxy in `serverless.yml`.
+
+```yaml
+custom:
+  apiGatewayServiceProxies:
+    - dynamodb:
+        path: /dynamodb
+        method: post
+        tableName: { Ref: 'YourTable' }
+        hashKey: 'id' # You must define hashKey attribute with `post` method
```

Comment:
does this mean the hash key is the string literal `id`?

Reply:
Also, reading this (without looking at the implementation) it's not clear to me if `id` is the attribute name for the hash key, or the attribute value to be used as hash key.